gamemode/config/sh_config.lua

```diff
@@ -105,7 +105,11 @@ end, {
 })
 ix.config.Add("runSpeed", 235, "How fast a player normally runs.", function(oldValue, newValue)
 	for _, v in ipairs(player.GetAll())	do
-		v:SetRunSpeed(newValue)

```

Comment:
This only has to be done... because you are changing the movespeed calculation fired in CalcStaminaChange...